STATUS.md

```diff
@@ -4,28 +4,40 @@ Symbols:
 
 - 🚧 Work in Progress
 - 🚫 Not planned
-- ❓Mabe implement
+- ❓ Maybe implement
 
 ## Low Level API
 
-- [x] Paket Parser
-  - [x] Old Format (v3)
-  - [x] New Format (v4)
-  - [ ] 🚧 New New Format (v5)
+- [x] Packet Parser
+  - [x] Old Format ("v3")
+  - [x] New Format (RFC 2440, 4880) ("v4")
+  - [ ] draft-koch-librepgp Format ("v5")
+  - [ ] draft-ietf-openpgp-crypto-refresh Format ("v6")
+- [x] Packet Generation
 - [x] ASCII Armor
   - [x] Reading
   - [x] Writing
 - [ ] Cleartext Signature Framework
-- [x] Paket Generation
-- [x] Signatures
+- [x] Signatures (v4)
   - [x] Validation
   - [x] Generation
-- [x] Encryption
-- [x] Decryption
-- [x] Keys
+- [ ] Signatures (v6)
```

Comment:
I actually don't know, is there a "v5 signature"?

Reply:
draft-koch-librepgp specifies at least the following: a "v5 key" format, "v5 signature", v5 SKESK,  and a new packet type 20 "OCB Encrypted Data Packet" (for LibrePGP's flavor of AEAD encryption)

my idea for this PR was to only mention draft-koch-librepgp at the top, and leave it at that, for now.